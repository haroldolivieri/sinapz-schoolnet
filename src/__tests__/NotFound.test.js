import React from 'react'
import NotFound from '../components/NotFound'
import { shallow } from 'enzyme'

describe('NotFound', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<NotFound />)
        expect(wrapper).toMatchSnapshot()
    })
})
