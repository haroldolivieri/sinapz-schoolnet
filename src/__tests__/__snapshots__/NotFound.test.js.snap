// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<div>
  <h1>
    Not Found
  </h1>
</div>
`;
